tests/unit/database_cache_backend_test.py

```diff
@@ -0,0 +1,35 @@
+import time
+
+import pytest
+
+from caching.backends import DatabaseBackend
+
+
+@pytest.mark.asyncio
+async def test_backend(tmpdir):
+    backend = DatabaseBackend(url=f"sqlite:///{tmpdir}/test.db")
+    await backend.setup()
+
+    assert await backend.get("foo") is None
```

Comment:
I know foo and bar are common among IT people, but personally, I find them confusing, because they have no meaning - which is also what they are for. Personally, I would prefer something like "dummy_key" and "dummy_value" or similar. This is just my opinion and not a task for you to change. I just wanted to share my thoughts here and why I will probably refrain from using foo bar myself. Furthermore we had a customer in the past who was really confused by foobar, because he came from aviation. There, they use FUBAR = fucked up beyond all recognition.  :D

Reply:
Good point - I usually use meaningful variable names wherever possible :-)

Reply:
Great, thank you :)

Reply:
done